scripts/node_install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+CMAKE_OPTIONS=${CMAKE_OPTIONS:-"-DCMAKE_BUILD_TYPE=Release -DENABLE_NODE_BINDINGS=On -DENABLE_MASON=On"}
```

Comment:
Why do we force a mason build? Because it's the only way to be sure users have all the deps.?